skaffold.yaml

```diff
@@ -74,7 +75,23 @@ profiles:
       - k8-manifests/dev/api-server.yaml
       - k8-manifests/dev/inventory.yaml
       - k8-manifests/dev/payments.yaml
+- name: publish
```

Comment:
Is it a common use-case to use Skaffold modules like this? (e.g. a different module for "deploy" and for "build" steps).
Why not put both the build and deploy steps in the same profile and use different built-in flags for `skaffold` depending on if you just want to build, or if you want to build+deploy?

Reply:
Here's an example with Bank of Anthos: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/skaffold.yaml#L47-L77

Reply:
We also have the same pattern (as in BoA) in this repo. Our use case was that we wanted to only build the images and not publish them anywhere. I see that using `build` command does exactly that. I think we must have missed it. 

Let me check again on this and get back!

Reply:
I checked locally, and when I do: `skaffold build -p dev --default-repo=<AR_REPO/GCR_REPO> -t <TAG>`

it seems like the local changes are built and published to the repository _(with tagging)_ and **not deployed to the cluster. @mike-ensor can we use it for your use case? Let us know.

Reply:
@bourgeoisor - reverted the changes and added a guide as to how to only build without deploy. Thank you for raising this.